src/main/java/com/revature/studyforce/flashcard/service/FlashcardService.java

```diff
@@ -0,0 +1,224 @@
+package com.revature.studyforce.flashcard.service;
+
+import com.revature.studyforce.flashcard.dto.FlashcardAllDTO;
+import com.revature.studyforce.flashcard.dto.FlashcardDTO;
+import com.revature.studyforce.flashcard.dto.NewFlashcardDTO;
+import com.revature.studyforce.flashcard.model.Difficulty;
+import com.revature.studyforce.flashcard.model.Flashcard;
+import com.revature.studyforce.flashcard.model.Topic;
+import com.revature.studyforce.flashcard.repository.FlashcardRepository;
+import com.revature.studyforce.flashcard.repository.TopicRepository;
+import com.revature.studyforce.user.model.User;
+import com.revature.studyforce.user.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.util.Locale;
+import java.util.Optional;
+
+/**
+ * Services for the Flashcard repository
+ * @author Luke
+ */
+@Service
+public class FlashcardService implements AbstractService {
```

Comment:
missing class level {@link}
Additionally all methods are missing specific {@link} to methods they are dependent on